Wikipedia/Code/SearchResultsViewController.swift

```diff
@@ -102,11 +102,11 @@ class SearchResultsViewController: ArticleCollectionViewController {
         }
         cell.configureForCompactList(at: indexPath.item)
         cell.setTitleHTML(result.displayTitleHTML, boldedString: resultsInfo?.searchTerm)
-       
         cell.articleSemanticContentAttribute = MWKLanguageLinkController.semanticContentAttribute(forContentLanguageCode: contentLanguageCode)
         cell.titleLabel.accessibilityLanguage = languageCode
         cell.descriptionLabel.text = descriptionForSearchResult(result)
         cell.descriptionLabel.accessibilityLanguage = languageCode
+        cell.actions = availableActions(at: indexPath)
```

Comment:
Could we change this to a pattern used elsewhere in the app?
```suggestion
        editController.configureSwipeableCell(cell, forItemAt: indexPath, layoutOnly: layoutOnly)
```

Reply:
@mcleinman Excellent, I've made that change!